azureeyemodule/app/model/ssd.cpp

```diff
@@ -20,15 +21,46 @@
 #include "../util/helper.hpp"
 #include "../util/labels.hpp"
 
+namespace custom {
+
+using GDetections = cv::GArray<cv::Rect>;
+using GPrims      = cv::GArray<cv::gapi::wip::draw::Prim>;
+
+G_API_OP(BBoxes, <GPrims(GDetections)>, "sample.custom.b-boxes") {
+    static cv::GArrayDesc outMeta(const cv::GArrayDesc &) {
+        return cv::empty_array_desc();
+    }
+};
+
+GAPI_OCV_KERNEL(OCVBBoxes, BBoxes) {
+    // Converts the rectangles into G-API's rendering primitives
+    static void run(const std::vector<cv::Rect> &in_face_rcs,
+                          std::vector<cv::gapi::wip::draw::Prim> &out_prims) {
+    }
+};
+
+} // namespace custom
 
 namespace model {
 
 /** An SSD network takes a single input and outputs a single output (which we will parse into boxes, labels, and confidences) */
 G_API_NET(SSDNetwork, <cv::GMat(cv::GMat)>, "ssd-network");
 
-SSDModel::SSDModel(const std::string &labelfpath, const std::vector<std::string> &modelfpaths, const std::string &mvcmd, const std::string &videofile, const cv::gapi::mx::Camera::Mode &resolution)
+bool check_file_exist(std::string filepath)
```

Comment:
Double check that we don't already have a function that does this in the helper module (util/helper.[c/h]pp). If we don't, please move this function to that file, so other modules can use it.

Reply:
there is an existing one. use that instead